bundle/manifests/elasticsearch-operator.clusterserviceversion.yaml

```diff
@@ -115,7 +115,7 @@ metadata:
     operatorframework.io/arch.amd64: supported
     operatorframework.io/arch.ppc64le: supported
     operatorframework.io/arch.s390x: supported
-  name: elasticsearch-operator.v5.1.0
+  name: elasticsearch-operator.v5.2.0
```

Comment:
@vimalk78 do we need to set ` olm.skipRange: '>=4.6.0-0 <5.1.0'` to 5.3.0 on the max end

Reply:
we don't use anything from `bundle/` for CPaaS IIRC